packages/actions/scoreboard/src/components/Scoreboard.js

```diff
@@ -0,0 +1,68 @@
+import PropTypes from 'prop-types'
+import {useEffect} from 'react'
+import {FormattedMessage} from 'react-intl'
+import styled from 'styled-components'
+import {LoadingSpinner, Typography} from 'tocco-ui'
+
+const StyledContainer = styled.div`
+  width: 148px;
+  height: 148px;
+  position: relative;
+  align-self: center;
+`
+
+const StyledContent = styled.div`
+  margin: 0;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+`
+const Scoreboard = ({data, fetchData}) => {
+  useEffect(() => {
+    fetchData()
+  }, [fetchData])
+
+  let content
+
+  if (data === undefined) {
+    content = <LoadingSpinner />
+  } else if (data === null) {
+    content = (
+      <Typography.P>
+        <FormattedMessage id="client.scoreboard.fetchFailed" />
+      </Typography.P>
+    )
+  } else {
+    content = data.map(entry => {
+      const defaultDisplay = <p>{entry.participantBean.defaultDisplay}</p>
+      const rank = <p>Ranking: {entry.ranking}</p>
+      const points = <p>Points: {entry.points}</p>
+
+      const returnValue = [defaultDisplay, rank, points]
+      return returnValue
+    })
```

Comment:
suggestion(non-blocking): usually we would write this in jsx to have a clearer understanding of the output

e.g.
```
content = data.map(({ranking, points, participantBean}) => (
      <React.Fragment key={participantBean.key}>
        <p>{participantBean.defaultDisplay}</p>
        <p>Ranking: {entry.ranking}</p>
        <p>Points: {entry.points}</p>
      <React.Fragment/>
    )}
```

React.Fragment is needed when having multiple jsx-elements: https://reactjs.org/docs/fragments.html
`key` prop is needed for arrays: https://reactjs.org/docs/lists-and-keys.html#keys